src/components/Breadcrumbs/BreadcrumbItem.tsx

```diff
@@ -31,9 +31,10 @@ const BreadcrumbItem: React.FC<BreadcrumbItemProps> = ({
 }) => {
   const hasIcon = isNotUndefined(iconName);
   const label = hasIcon ? { 'aria-label': children } : { children };
+  const isText = isNilOrEmpty(to) && isNilOrEmpty(href);
   return (
     <ListItem>
-      {isSelected ? (
+      {isSelected || isText ? (
```

Comment:
I guess we can remove the `isSelected` property since that was filled in the `cloneElement` and the implementation was exactly the same as the `isText`.